Configuration/StandardSequences/python/AlCaHarvesting_cff.py

```diff
@@ -225,7 +225,7 @@
 EcalPedestals  = cms.Path(ALCAHARVESTEcalPedestals)
 SiStripGainsAAG = cms.Path(ALCAHARVESTSiStripGainsAAG)
 LumiPCC = cms.Path(ALCAHARVESTLumiPCC)
-SiPixelQuality = cms.Path(ALCAHARVESTSiPixelQuality)
+SiPixelQuality = cms.Path(ALCAHARVESTSiPixelQuality)#+siPixelPhase1DQMHarvester)
```

Comment:
@tocheng @lpernie this statement change is effectively doing nothing, I understand it is a placeholder left for the future, but what is the plan here?

Reply:
@tocheng @lpernie @arunhep could you please clarify?

Reply:
The original plan is to use the SiPixel Phase-I DQM Harvester class (as a template) to normalize the bincontent of the track map plots (in tracker DQM output) by the total number of lumi sections. So the bin content will represent the fraction of time that the ROC is bad in a run. (Then it will be easier to identify new ROCs that are bad for during whole run)
But I realize it needs a lot of customization as the functionalities I need is protect by the SiPixel DQM framework. I contacted the tracker DQM conveners but they are too busy to work on this special request.
So I decide to put this development for the future. The plots with binconent as the absolute number of lumi section is still good enough to monitor the payloads and identify new bad ROCs.
Currently siPixelPhase1DQMHarvester is dummy for the SiPixelQuality DQM production.
I commented it to prevent the impact from potential central SiPixel DQM development in the future.
Hopefully this clarifies the change.

Reply:
@tocheng thank you for the detailed discussion. At this point anyway I need to understand whether this development is really needed in 10_2_0, as apparently stated by @lpernie @arunhep in the ORP spreadsheet, or it can stay on hold, as effectively it is dummy so far, as I get from your explanation

Reply:
@fabiocos I think it is critical for this PR to be merged into CMSSW_10_2_0. 
Because it fix a bug for the SiPixelQuality conditions to be used for prompt reconstruction.
The DQM monitoring in the PR provides the feature to monitor the SiPixelQuality conditions and also help offline tracker DQM shifter to find new bad components.
They are very critical for the data taking in CMSSW_10_2_0 release.